{"timestamp":"2025-10-05T12:20:18.049990Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-05T12:20:18.051702Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/json_to_postgres.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-05T12:20:18.078246Z","level":"warning","event":"The `airflow.operators.python.PythonOperator` attribute is deprecated. Please use `'airflow.providers.standard.operators.python.PythonOperator'`.","category":"DeprecatedImportWarning","filename":"/opt/airflow/dags/json_to_postgres.py","lineno":6,"logger":"py.warnings"}
{"timestamp":"2025-10-05T12:20:18.102760Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.104226Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.105528Z","level":"info","event":"Current task name:load_raw_json_task","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.106749Z","level":"info","event":"Dag name:json_to_postgres_monthly_new","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.125849Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":994,"error_detail":[{"exc_type":"OperationalError","exc_value":"(psycopg2.OperationalError) connection to server at \"postgres\" (172.18.0.2), port 5432 failed: FATAL:  database \"dwh\" does not exist\n\n(Background on this error at: https://sqlalche.me/e/14/e3q8)","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":82,"name":"run"},{"filename":"/opt/airflow/dags/json_to_postgres.py","lineno":73,"name":"load_raw_json"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":3153,"name":"begin"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":3325,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":96,"name":"__init__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":3404,"name":"raw_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":3374,"name":"_wrap_pool_connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":2208,"name":"_handle_dbapi_exception_noconnection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py","lineno":211,"name":"raise_"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":3371,"name":"_wrap_pool_connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":327,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":894,"name":"_checkout"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":493,"name":"checkout"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py","lineno":145,"name":"_do_get"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py","lineno":70,"name":"__exit__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py","lineno":211,"name":"raise_"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py","lineno":143,"name":"_do_get"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":273,"name":"_create_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":388,"name":"__init__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":690,"name":"__connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py","lineno":70,"name":"__exit__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py","lineno":211,"name":"raise_"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":686,"name":"__connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/create.py","lineno":574,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/default.py","lineno":598,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py","lineno":122,"name":"connect"}],"is_group":false,"exceptions":[]},{"exc_type":"OperationalError","exc_value":"connection to server at \"postgres\" (172.18.0.2), port 5432 failed: FATAL:  database \"dwh\" does not exist\n","exc_notes":[],"syntax_error":null,"is_cause":true,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":3371,"name":"_wrap_pool_connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":327,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":894,"name":"_checkout"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":493,"name":"checkout"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py","lineno":145,"name":"_do_get"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py","lineno":70,"name":"__exit__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py","lineno":211,"name":"raise_"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/impl.py","lineno":143,"name":"_do_get"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":273,"name":"_create_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":388,"name":"__init__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":690,"name":"__connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/langhelpers.py","lineno":70,"name":"__exit__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py","lineno":211,"name":"raise_"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/pool/base.py","lineno":686,"name":"__connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/create.py","lineno":574,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/default.py","lineno":598,"name":"connect"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/psycopg2/__init__.py","lineno":122,"name":"connect"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-05T12:20:18.149012Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.150161Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.151215Z","level":"info","event":"Task:<Task(PythonOperator): load_raw_json_task>","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.152467Z","level":"info","event":"Failure caused by (psycopg2.OperationalError) connection to server at \"postgres\" (172.18.0.2), port 5432 failed: FATAL:  database \"dwh\" does not exist","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.153495Z","level":"info","event":"","logger":"task.stdout"}
{"timestamp":"2025-10-05T12:20:18.154699Z","level":"info","event":"(Background on this error at: https://sqlalche.me/e/14/e3q8)","logger":"task.stdout"}
